frontend/src/reports/ReportProvider.tsx

```diff
@@ -107,33 +108,94 @@ function ReportProvider(props: { madLib: MadLib; setMadLib: Function }) {
 
   return (
     <>
-      <Dialog
-        open={shareModalOpen}
-        onClose={() => setShareModalOpen(false)}
-        aria-labelledby="alert-dialog-title"
-        aria-describedby="alert-dialog-description"
-      >
-        <DialogTitle id="alert-dialog-title">Link to this Report</DialogTitle>
-        <DialogContent>
-          <DialogContentText id="alert-dialog-description">
-            {getMadLibPhraseText(props.madLib)}
-          </DialogContentText>
-          <DialogContentText id="alert-dialog-description">
-            {linkToMadLib(props.madLib.id, props.madLib.activeSelections, true)}
-          </DialogContentText>
-        </DialogContent>
-      </Dialog>
-      <div className={styles.ReportToolbar}>
-        <Button
-          color="primary"
-          startIcon={<ShareIcon />}
-          onClick={() => setShareModalOpen(true)}
-          data-tip="Share a Link to this Report"
+      <div className={styles.ReportWrapper}>
+        <Dialog
+          open={shareModalOpen}
+          onClose={() => setShareModalOpen(false)}
+          aria-labelledby="alert-dialog-title"
+          aria-describedby="alert-dialog-description"
         >
-          Share
+          <DialogTitle id="alert-dialog-title">Link to this Report</DialogTitle>
+          <DialogContent>
+            <DialogContentText id="alert-dialog-description">
+              {getMadLibPhraseText(props.madLib)}
+            </DialogContentText>
+            <DialogContentText id="alert-dialog-description">
+              {linkToMadLib(
+                props.madLib.id,
+                props.madLib.activeSelections,
+                true
+              )}
+            </DialogContentText>
+          </DialogContent>
+        </Dialog>
+        <div className={styles.ReportToolbar}>
+          <Button
+            color="primary"
+            startIcon={<ShareIcon />}
+            onClick={() => setShareModalOpen(true)}
+            data-tip="Share a Link to this Report"
+          >
+            Share
+          </Button>
+        </div>
+        {getReport()}
+      </div>
+      <div className={styles.MissingDataInfo}>
+        <h1>What Data Are Missing?</h1>
+        <p>
+          In this tracker, we are using <a href="/">COVID Tracking Project</a>,{" "}
+          <a href="/">CDC Public Datasets</a>,{" "}
+          <a href="/">CREW Data American</a>, and{" "}
+          <a href="/">American Community Survey data</a>. Some soures are more
```

Comment:
Why list these manually? Can we automatically extract these from the metadata? (I know the metadata is fake but we have to fix it anyway)

Reply:
I would defer to the copywriters- not sure if the names listed here are exactly the same as elsewhere. I've made a TODO